src/common.c

```diff
@@ -544,8 +544,8 @@ int fi_epoll_wait(struct fi_epoll *ep, void **contexts, int max_contexts,
 	int found = 0;
 
 	ret = poll(ep->fds, ep->nfds, timeout);
-	if (ret == -1)
-		return -errno;
+	if (ret == SOCKET_ERROR)
+		return -ofi_sockerr();
```

Comment:
Here and above poll is used with sockerr.  Is this correct for windows?

Reply:
Yes, it's correct. Because poll -> WSAPoll for Windows. And set value in errno should be checked via WSAGetLastError()